src/components/molecules/event/eventRegistrationQR/EventRegistrationQR.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Event } from 'models/apiModels';
+import Button from 'components/atoms/button/Button';
+
+import { baseUrl } from 'constants/apiConstants';
+import { createQR } from 'api';
+import { useToast } from 'hooks/useToast';
+
+const EventRegistrationQR: React.FC<{ event: Event }> = ({ event }) => {
+  const { addToast } = useToast();
+  /* Run get request for event QR code */
+  const getEventQR = async (url: string): Promise<void> => {
+    return new Promise<void>((resolve, reject) => {
+      const filename = event.title + '.pdf';
+      const request = new Request(baseUrl + 'event/' + url + '/qr', {
```

Comment:
Can we change the `get` function to accept pdf as a content type so we can use that?